diff --git a/cc/library.go b/cc/library.go
index 4a173a5..f875a2a 100644
--- a/cc/library.go
+++ b/cc/library.go
@@ -363,7 +363,7 @@ func (library *libraryDecorator) compile(ctx ModuleContext, flags Flags, deps Pa
 		total_length := len(library.baseCompiler.Properties.Srcs) + len(deps.GeneratedSources) + len(library.Properties.Shared.Srcs) +
 			len(library.Properties.Static.Srcs)
 		if total_length > 0 {
-			flags.SAbiDump = true
+			flags.SAbiDump = false
 		}
 	}
 	objs := library.baseCompiler.compile(ctx, flags, deps)
